acos_client/v30/slb/virtual_port.py

```diff
@@ -67,15 +67,11 @@ def get(self, virtual_server_name, name, protocol, port):
         )
         return self._get(url)
 
-    def _set(self, virtual_server_name, name, protocol, port,
-             service_group_name,
-             s_pers_name=None, c_pers_name=None, stats=0, update=False,
-             no_dest_nat=None,
-             exclude_minimize=[],
-             autosnat=False,
-             ipinip=False,
-             pool=None,
-             **kwargs):
+    def _set(self, virtual_server_name, name, protocol, port, service_group_name, s_pers_name=None, c_pers_name=None,
```

Comment:
I think `stats` got changed and needs to be changed back. Or was that a fix? I am confused.

Reply:
It should have been status I think, because stats here doesn't make sense.  It should be consistent throughout.